### ISSUE TITLE
NTR: Sclerotic Bone lesion

### ISSUE THREAD

**Issue by Daniel-Olson** [obophenotype/human-phenotype-ontology#10006]
**Preferred term label:**
Sclerotic Bone lesion

**Synonyms**
Sclerotic lesion of a bone

**Definition (free text, please give PubMed ID)**
an unusual hardening or thickening of a bone.

**Parent term (use hpo.jax.org/app)**
Abnormal bone structure (HP:0003330)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-8134-1207

**Comment by pnrobinson** [comment]
@Daniel-Olson 
there are already numerous terms with this meaning, see 
https://hpo.jax.org/app/browse/term/HP:0011001
and descendants.
Could you provide more context?
